### DISCUSSION TITLE
Font size discrepancy in saved plot using ggsave: nested axis text size not change after saving

### DISCUSSION THREAD

**Discussion by SyedTuhinAli** [teunbrand/ggh4x#136] [General]
p= ggplot(df, 
       aes(x = interaction(comb_spp, comb_treat), y = LMF)) +
  geom_boxplot(aes(fill = comb_spp)) +
  scale_x_discrete(guide = guide_axis_nested(delim = ".")) +
theme(ggh4x.axis.nesttext.x = element_text(size = 16)) 

works on the R output plot however after saving ggsave("output.jpeg", p, width = 12, height = 8, dpi= 300) the nested x axis text size is no longer 16.

**Comment by teunbrand** [comment]
Does all other text scale just fine? I just quickly tried the first example in `?guide_axis_nested` with size = 16, but it doesn't look off to my eyes.

**Reply by SyedTuhinAli** [reply]
well in the r Plots, it does change, however, whenever saved as jpeg or png, other than pdf, it does not change. could you please check it again?

**Reply by teunbrand** [reply]
Yeah I checked by saving as jpeg, this is what it looks like on my end:
![test](https://github.com/teunbrand/ggh4x/assets/49372158/5288f84e-3c1a-4c9b-aedf-46dcb55ae8d7)

**Reply by SyedTuhinAli** [reply]
yes, it is working. thanks. I had to increase the size quite a bit, for example, 40.
